bin/setup

```diff
@@ -25,6 +25,11 @@ FileUtils.chdir APP_ROOT do
     FileUtils.cp 'config/database.yml.sample', 'config/database.yml'
   end
 
+  unless File.exists?('storage/disposable_email_blocklist.conf')
```

Comment:
Please just delete this, let's tweak the reader to be forgiving if the dir/file doesn't exist. We can't count on all developers to run this script, especially if they already have a dev env set up.